rocprim/include/rocprim/block/block_shuffle.hpp

```diff
@@ -156,6 +156,16 @@ class block_shuffle
         );
     }
 
+    /// \brief Shuffles data across threads in a block, offseted by the distance value.
+    ///
+    /// \par A thread with  threadId i receives data from a thread with threadIdx (i-distance), whre distance may be a negative value.
```

Comment:
looks like an extra whitespace between with and threadId.
threadId should be threadIdx to be consistent with subsequent usage of threadIdx.  Or change threadIdx to  threadId, whatever seems more appropriate.

Reply:
That makes sense, thanks. I've fixed this up, along with the documentation for the overloads (where this documentation was copied from), since they contain the same typos.